### ISSUE TITLE
Login screen, Forgot Password screen and Dashboard screen has to fit properly on a standard laptop screen.

### ISSUE THREAD

**Issue by onepointpb** [OnePointGlobal/OnePoint-Global-Account-Toolkit#64]
- Login screen, Forgot Password screen, overlay and Dashboard screen has to fit properly in a standard laptop screen. They should not scroll.
- Make the next and back arrow on the overlay page bit smaller as suggested by the design team.

**Comment by dougmorris** [comment]
Further to these comments, here is some feedback:

1.	The login fields are large and the font is too small, it looks poor. I previously requested that we use the google inbox font if we are mimicking their technology. This font is too small and too skinny, it looks out of place in that large field
2.	The logo, title, and fields could be moved up a bit as its all crammed at the bottom and makes the picture etc. overlap on the support, about us fields at the bottom
3.	I have to view at 75% screen size to prevent scrolling
![image](https://cloud.githubusercontent.com/assets/4094389/26299397/9de3b736-3ed2-11e7-807f-7c16869b53a5.png)

**Comment by pavankumarkolla** [comment]
Resolved in Dev

**Comment by onepointpb** [comment]
1. Edit Profile and Change Password screen do not fit on the laptop screen. 
2. Next and back arrow is fixed as per the Design team's suggestion. But the overlay close button is now changed.

**Comment by deepika-Gopala** [comment]
1. Considering the pixels of the design and number of fields in the profile page , it cannot be fit to the smallest of the laptop screen , this will affect the vertical spacing between the filed and they will look crunched , Mainly because the  header and the footer are already fixed and are covering the 15% of the screen.

Technically the laptop that's being used to test is  is lesser than the standard size that's been considered for the development which is 1024*768 and the one tested in browser is 1233*676 which is below the range. So the scroll is expected in this case.

**Comment by onepointpb** [comment]
So, could you please let me know for which laptop screen this will work?

**Comment by deepika-Gopala** [comment]
as mentioned it will work for the screen >= 1024 * 768

**Comment by onepointpb** [comment]
For change password screen there is no content left at the bottom, but still the page is scrolling. Is it because of the reason you mentioned above?

**Comment by deepika-Gopala** [comment]
yeah its mainly because of that ad however we have discussed with design team and they ill be providing a smaller image for the key and the problem will be sorted.

**Comment by premonepoint** [comment]
Fixed

**Comment by onepointpb** [comment]
The change Password screen is not yet fixed.

**Comment by veereshms** [comment]
Change password screen is still displayed with scroll bar, kindly discuss this with design team.
